terraform/helm/aptos-node/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,72 @@
+{{- range $i, $e := until (int .Values.numValidators) }}
```

Comment:
wow, for loops in helm are really ugly lol

Reply:
yea... hoepfully it's readable enough though